client/src/pages/learn/2022/responsive-web-design/learn-css-animation-by-building-a-ferris-wheel/index.md

```diff
@@ -1,9 +1,32 @@
 ---
-title: Introduction to the Learn CSS Animation by Building a Ferris Wheel Project
+title: Introduction to the Learn CSS Animation by Building a Ferris Wheel
 block: learn-css-animation-by-building-a-ferris-wheel
 superBlock: 2022/responsive-web-design
 ---
 
-## Introduction to the Learn CSS Animation by Building a Ferris Wheel Project
+## Introduction to the Learn CSS Animation by Building a Ferris Wheel
 
-This is a test for the new project-based curriculum.
+Have you ever wanted to bring your web pages to life with fun and interactive animations? CSS animation allows you to add movement and interactivity to your web pages, making them more engaging and dynamic. It’s like adding a sprinkle of magic to your design!
+
+You will learn techniques like using CSS `keyframes` to control how your elements behave.
+
+```css
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
```

Comment:
in the project we use only percentages, not `from` and `to`

Reply:
Yeah there's going to be a lot to improve with the copy for these. I'm just trying to get a basic starting point for now.